plugins/kubernetes/src/components/KubernetesContent/KubernetesContent.tsx

```diff
@@ -45,27 +85,22 @@ export const KubernetesContent: FC<{ entity: Entity }> = ({ entity }) => {
     <Page theme={pageTheme.tool}>
       <Content>
         <Grid container spacing={3} direction="column">
-          {kubernetesObjects === undefined && <div>loading....</div>}
+          {kubernetesObjects === undefined && <Progress />}
           {error !== undefined && <div>{error}</div>}
-          {kubernetesObjects !== undefined && (
-            <div>
-              {kubernetesObjects.items.map((item, i) => (
-                <Grid item key={i}>
-                  <InfoCard key={item.cluster.name} title={item.cluster.name}>
-                    {item.resources.map((fr, j) => (
-                      <div key={j}>
-                        <br />
-                        {fr.type}:{' '}
-                        {(fr.resources as any)
-                          .map((v: any) => v.metadata.name)
-                          .join(' ')}
-                      </div>
-                    ))}
-                  </InfoCard>
-                </Grid>
-              ))}
-            </div>
-          )}
+          {kubernetesObjects !== undefined &&
```

Comment:
Might be nicer to `kubernetesObjects?.items.map(...)` here

Reply:
sounds good